> D:\a\1\s\src\System.Management.Automation\SourceGenerators\PSVersionInfoGenerator\PSVersionInfoGenerator.cs(14,18): warning RS1036: 'SMA.PSVersionInfoGenerator': A project containing analyzers or source generators should specify the property '<EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>' [D:\a\1\s\src\System.Management.Automation\SourceGenerators\PSVersionInfoGenerator\PSVersionInfoGenerator.csproj]

I guess this asks to add `<EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>` in PSVersionInfoGenerator.csproj 

> D:\a\1\s\src\Microsoft.Management.UI.Internal\ShowCommand\Controls\ParameterSetControl.xaml.cs(164,112): error CS1501: No overload for method 'Create' takes 1 arguments [D:\a\1\s\src\Microsoft.Management.UI.Internal\Microsoft.PowerShell.GraphicalHost_l0ynpe1u_wpftmp.csproj]

Please fix.

--- a/src/Microsoft.Management.Infrastructure.CimCmdlets/CimSessionProxy.cs
+++ b/src/Microsoft.Management.Infrastructure.CimCmdlets/CimSessionProxy.cs
@@ -875,7 +875,7 @@ namespace Microsoft.Management.Infrastructure.CimCmdlets
                         parameters.Append(',');
                     }
 
-                    parameters.AppendFormat(CultureInfo.CurrentUICulture, $@"'{key}' = {parameterList[key]}");
+                    parameters.Append(CultureInfo.CurrentUICulture, $@"'{key}' = {parameterList[key]}");
                 }
             }
 

--- a/src/Microsoft.Management.UI.Internal/ShowCommand/Controls/MultipleSelectionControl.xaml.cs
+++ b/src/Microsoft.Management.UI.Internal/ShowCommand/Controls/MultipleSelectionControl.xaml.cs
@@ -43,7 +43,7 @@ namespace Microsoft.PowerShell.Commands.ShowCommandInternal
 
             foreach (object selectedItem in multipleSelectionDialog.listboxParameter.SelectedItems)
             {
-                newComboText.AppendFormat(CultureInfo.InvariantCulture, $"{selectedItem},");
+                newComboText.Append(CultureInfo.InvariantCulture, $"{selectedItem},");
             }
 
             if (newComboText.Length > 1)

--- a/src/Microsoft.Management.UI.Internal/ShowCommand/Controls/ParameterSetControl.xaml.cs
+++ b/src/Microsoft.Management.UI.Internal/ShowCommand/Controls/ParameterSetControl.xaml.cs
@@ -161,7 +161,7 @@ namespace Microsoft.PowerShell.Commands.ShowCommandInternal
             multiControls.comboxParameter.SetBinding(ComboBox.TextProperty, valueBinding);
 
             // Add AutomationProperties.AutomationId for Ui Automation test.
-            multiControls.SetValue(System.Windows.Automation.AutomationProperties.AutomationIdProperty, string.Create($"combox{parameterViewModel.Name}"));
+            multiControls.SetValue(System.Windows.Automation.AutomationProperties.AutomationIdProperty, string.Create(CultureInfo.CurrentCulture, $"combox{parameterViewModel.Name}"));
 
             multiControls.comboxParameter.SetValue(
                 System.Windows.Automation.AutomationProperties.NameProperty,

--- a/src/System.Management.Automation/SourceGenerators/PSVersionInfoGenerator/PSVersionInfoGenerator.csproj
+++ b/src/System.Management.Automation/SourceGenerators/PSVersionInfoGenerator/PSVersionInfoGenerator.csproj
@@ -9,6 +9,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <LangVersion>10.0</LangVersion>
     <SuppressNETCoreSdkPreviewMessage>true</SuppressNETCoreSdkPreviewMessage>
+    <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
   </PropertyGroup>
 
   <ItemGroup>

